src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.skia.cs

```diff
@@ -58,6 +71,74 @@ private enum ContextMenuItem
 
 	internal ContentControl ContentElement => _contentElement;
 
+	// We track what constitutes one typing "action" that can be undone/redone. The general gist is that
+	// any sequence of characters (with backspace allowed) without any navigation moves (pointer click, arrow keys, etc.)
+	// will be one "run"/"action". However, there are some arbitrary exceptions, so that is only a rule of thumb.
+	private bool CurrentlyTyping
```

Comment:
Let's make a method instead of a property, it will be more in-line with what the rest of the codebase uses for this, particularly because setting the value may end up not being done. Something like `TrySetCurrentlyTyping`.